### PR TITLE
Update contributing docs

### PR DESCRIPTION
**Pull request by mattwiller**
Drafted some updates to our documentation on contributing to the project, with a few minor aims:
- Streamline and align the doc pages and README content
- Reduce focus on code contributions and highlight the various ways people can participate
- Lower the activation energy for someone interested to make their first contribution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,35 +19,51 @@ Medplum is a developer platform that enables flexible and rapid development of h
...
+and [Acceptable Use Policies](https://help.github.com/en/github/site-policy/github-acceptable-use-policies). We expect
+discussions in issues and pull requests to stay positive, productive, and respectful. Remember: there are real people on
+the other side of the screen!
 
-#### Reporting a bug or discussing a feature idea
+#### Reporting a bug or proposing a new feature
 
-If you found a technical bug on Medplum or have ideas for features we should implement, the issue tracker is the best place to share your ideas. Make sure to follow the issue template and you should be golden! ([click here to open a new issue](https://github.com/medplum/medplum/issues/new))
+If you found a technical bug on Medplum or have ideas for features we should implement, the issue tracker is the best
+place to . ([click here to open a new issue](https://github.com/medplum/medplum/issues/new))
```
Comment (codyebberson) [inline README.md:44]: `place to share your ideas.`?

File: README.md [hunk 2]
```diff
@@ -19,35 +19,51 @@ Medplum is a developer platform that enables flexible and rapid development of h
...
+### Writing documentation or blog content
+
+Did you learn how to do something using Medplum that wasn't obvious on your first try? By contributing your new knowledge
+to our documentation, you cna help others who might have a similar use case!
+
+Our documentation is hosted on [medplum.com/docs](/docs), but it is built from [Markdown](https://www.markdownguide.org/)
+files in our [`docs` package](https://github.com/medplum/medplum/tree/main/packages/docs/docs).
+
+For relatively small changes, you can edit files directly from your web browser on [Github.dev](https://github.dev/medplum/medplum/blob/main/packages/docs/docs/home.md)
+without needing to clone the repository.
```
Comment (codyebberson) [inline README.md:55]: Consider referencing our "Contribute without coding" issue?  https://github.com/medplum/medplum/issues/2249

File: packages/docs/docs/contributing/contribute-without-coding.md
Comment (codyebberson) [inline packages/docs/docs/contributing/contribute-without-coding.md:1]: @reshmakh  - any objections to deleting this file?  any concerns about broken URLs?

File: packages/docs/docs/contributing/clone-the-repo.md
Comment (codyebberson) [inline packages/docs/docs/contributing/clone-the-repo.md:1]: @reshmakh  - any objections to deleting this file?  any concerns about broken URLs?

Comment (reshmakh) [reply packages/docs/docs/contributing/clone-the-repo.md:1]: No objections.

File: README.md (hunk 2 above)
Comment (mattwiller) [reply README.md:55]: Most of the copy in that issue is reproduced in the Contributing docs page, but I'll link to that!

**Review by codyebberson — APPROVED** [review]

**Pull request merged**